src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -73,4 +73,153 @@ module.exports = class LevelPlane extends Array {
 
     return `rails${variant}`;
   }
+
+  checkConnectionOnPlace(substring, blockType, LevelModel) {
```

Comment:
If we're going this route, please pass in `levelModel` to the constructor.  The level plane instance will always be associated with the same model.

Reply:
Roger that. I pass in the LevelModel when the function is called, so it's using that argument, but I added it to the constructor as well.

Reply:
Why both?  Once it's passed into the constructor, saving `this.levelModel = levelModel` will make it available as an instance variable.

Reply:
Yeah, I see what you mean. I didn't register that the reference on construction would be just that: a reference, rather than copying a snapshop of the model. I have this working with your suggestion